File: backend/src/main/java/com/easyone/travelance/domain/travel/dto/NoticeAllResponseDto.java
package com.easyone.travelance.domain.travel.dto;

import com.easyone.travelance.domain.travel.entity.Notice;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class NoticeAllResponseDto {
    private Long noticeId;
    private String title;
    private String content;
    private String link;

    public NoticeAllResponseDto(Notice notice) {
        this.noticeId = notice.getId();
        this.title = notice.getTitle();
        this.content = notice.getContent();
        this.link = notice.getLink();
    }
}

File: Travelance/data/src/main/java/com/moneyminions/data/model/traveldone/response/NoticeAllResponseDto.kt
package com.moneyminions.data.model.traveldone.response


import com.google.gson.annotations.SerializedName

data class NoticeAllResponseDto(
    @SerializedName("content")
    val content: String,
    @SerializedName("link")
    val link: String,
    @SerializedName("noticeId")
    val noticeId: Int,
    @SerializedName("title")
    val title: String
)